front/app/containers/ProjectsShowPage/timeline/Ideas.tsx

```diff
@@ -79,38 +82,58 @@ const IdeasContainer = ({ projectId, phase, className }: InnerProps) => {
       id="project-ideas"
       className={`e2e-timeline-project-idea-cards ${className || ''}`}
     >
-      {!isVotingContext && (
-        <Title variant="h2" mt="0px" mb="20px" color="tenantText">
-          <FormattedMessage
-            {...getInputTermMessage(inputTerm, {
-              idea: messages.ideas,
-              option: messages.options,
-              project: messages.projects,
-              question: messages.questions,
-              issue: messages.issues,
-              contribution: messages.contributions,
-              proposal: messages.proposals,
-              initiative: messages.initiatives,
-              petition: messages.petitions,
-            })}
+      {isVotingContext ? (
+        <IdeaCardsWithoutFiltersSidebar
```

Comment:
These have many shared props, so you can make a sharedProps object for them. I'll put up a PR for this.

Reply:
https://github.com/CitizenLabDotCo/citizenlab/pull/9492